content/en/docs/k8s-operator/automatic.md

```diff
@@ -112,7 +112,68 @@ Coming Soon
 
 ### Java
 
-Coming Soon
+The following command will create a basic Instrumentation resource that is
+configured for instrumenting Java services.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: demo-instrumentation
+spec:
+  exporter:
+    endpoint: http://demo-collector:4317
+  propagators:
+    - tracecontext
+    - baggage
+  sampler:
+    type: parentbased_traceidratio
+    argument: "1"
+EOF
+```
+
+By default, the Instrumentation resource that auto-instruments Java services
+uses `otlp` with the `grpc` protocol. This means that the configured endpoint
+must be able to receive OTLP over `grpc`. Therefore, the example uses
+`http://demo-collector:4317`, which will connect to the `grpc` port of the
```

Comment:
Make the link clickable / more-easily copiable?
```suggestion
<http://demo-collector:4317>, which will connect to the `grpc` port of the
```

Reply:
I don't think we want to make this clickable as it won't go anywhere useful.  Within the context of this page that address is only available from within your k8s cluster.